monero-outreach-docs/translations/es/merchants/how_to_accept_monero_es.md

```diff
@@ -0,0 +1,85 @@
+# Cómo aceptar Monero
+
+Si eres capaz de instalar una aplicación, serás capaz de configurar tu negocio para aceptar Monero (XMR). Esta guía explica las formas más comunes que los comerciantes usan para aceptar monero y cómo iniciarse de la forma menos sufrida y trabajar sobre ello a medida que creces.
+
+Lo primero que debes saber es que no hay ningún tipo de cuenta para comerciantes en Monero, solo necesitas una billetera y una dirección para empezar. Es recomendable que crees una dirección separada para tu negocio por las mismas razones que se sugiere tener separadas las cuentas corrientes personales y las de los negocios. Puedes usar el mismo software para la billetera, pero tener dedicadas unas direcciones para el negocio ayudará a mantener las cuentas organizadas.
+
+Si todavía no tienes una billetera o dirección de Monero, entonces querrás crear primero una. Las billeteras GUI/CLI oficiales de Monero de [getmonero.org/downloads](https://www.getmonero.org/downloads/) siempre serán la opción recomendada. Además, esta guía hará referencias a billeteras móviles, de las cuales puedes encontrar información en nuestra página [Monero Wallet Setup](https://www.monerooutreach.org/stories/monero-wallet-quickstart.html).
+
+### _La Anatomía de una Transacción de Monero_
+
+Las transacciones de Monero pasan por los siguientes pasos. No es necesario entender los aspectos técnicos de esto, pero como comerciante, necesitas tener un conocimiento básico de los pasos para ser capaz de decidir qué transacciones te vienen bien.
```

Comment:
I think it's more appropiate "intercambios" in that context instead of "compensaciones". If you are agree, I'll use that translation. If no, I'll follow your advice.

Reply:
ok with "intercambios"